01-getting-started/003-ecr-setup.md

```diff
@@ -74,6 +74,11 @@ kubectl -n <namespace_name> get secret example-team-ecr-credentials-output -o ya
 
 The values in kubernetes `Secrets` are always `base64` encoded so you will have to decode them before you can use them outside kubernetes. Inside the cluster, the nodes already have access to the ECR so you don't need to make any changes.
 
+This can be done at the command line using the following:
+```
+echo `echo QWxhZGRpbjpvcGVuIHNlc2FtZQ== | base64 --decode`
```

Comment:
This syntax seems a little confusing. If all you want is an extra carriage return at the end, how about this instead?

`echo QWxhZGRpbjpvcGVuIHNlc2FtZQ== | base64 --decode; echo`

Reply:
The other option is to provide a link to the [Authenticating with the repository](https://ministryofjustice.github.io/cloud-platform-user-docs/02-deploying-an-app/001-app-deploy/#authenticating-with-the-repository) in the next steps of this page?  

Not sure on the preferred flow of the steps?

Welcome back by the way! :)

Reply:
I've got another PR, #115 , which makes some changes to the walkthrough of deploying an application, so it's probably best to keep this as an example of using `base64` for now, in case the other change breaks the link.
(And thanks, it's good to be back).